_playbook/pdf.md

```diff
@@ -9,8 +9,23 @@ categories:
 
 ---
 
-Most PDFs simply  do not meet the Section 508 requirements. It is just simply so easy to produce PDFs, but so difficult to produce accessible ones. There are very few tools that allow you to tell the difference between.  One free tool and easy tool is the Tingtun Checker PDF Checker, but there are others that can provide more detailed results. 
+Most PDFs simply do not meet the Section 508 requirements. It is just simply so easy to produce PDFs, but so difficult to produce accessible ones. There are very few tools that allow you to tell the difference between. One free tool and easy tool is the [Tingtun Checker PDF Checker](http://checkers.eiii.eu/en/pdfcheck/), but does not show all errors that could be there. There are other tools that can provide more detailed results but require installation.
 
-It is possible to create PDF files that meet requirements of Section 508, but it is often overlooked or not done properly.  Often the source documents used to create them do not have proper semantics. It is always more expensive to try to remediate a PDF than to work to build accessibility best practices into the authoring tool.  
+It is possible to create PDF files that meet requirements of Section 508, but it is often overlooked or not done properly. Often the source documents used to create them do not have proper semantics. It is always more expensive to try to remediate a PDF than to implement accessibility best practices into the authoring tool.
 
-The UK’s Government Digital Services model decreases the reliance on PDFs and is actively working to move away from them online and pushing for HTML content instead. When publishing a PDF is unavoidable, ideally an HTML or MSWord version of the content should be available as well.
+The UK’s Government Digital Services model decreases the reliance on PDFs and is [actively working to move away from them online and pushing for HTML content instead](https://gds.blog.gov.uk/2018/07/16/why-gov-uk-content-should-be-published-in-html-and-not-pdf/). When publishing a PDF is unavoidable, ideally an HTML or MSWord version of the content should be available as well.
```

Comment:
@mgifford did you want to make the quote change? or should we adjust retext?

Reply:
It doesn't like "'", so what do we use instead? I suppose we could use "´" but that seems to be a pain. Single quotes are pretty much part of our language. How hard would it be to adjust retext? Is this something that should be pushed upstream?

Reply:
Actually, I think in this case it wants "'" instead of the smart quote in "...**UK´s** Government...". I prefer the regular quote.